lib/mrsk/cli/main.rb

```diff
@@ -107,8 +107,12 @@ def init
   desc "envify", "Create .env by evaluating .env.erb (or .env.staging.erb -> .env.staging when using -d staging)"
   def envify
     if destination = options[:destination]
+      File.write(".env.#{destination}", "")
+      File.chown(0600, ".env.#{destination}")
       File.write(".env.#{destination}", ERB.new(IO.read(Pathname.new(File.expand_path(".env.#{destination}.erb")))).result)
```

Comment:
Can pass `:perm` to set permissions if a file is created:
```suggestion
      File.write(".env.#{destination}", ERB.new(IO.read(Pathname.new(File.expand_path(".env.#{destination}.erb")))).result, perm: 0600)
```